site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Getting to know App resources better with kctrl"
+slug: kcrl-app-commands-blog
+date: 2022-04-28
+author: Soumik Majumder
+excerpt: "Get better visibility into your app resources with kctrl's new app commands and enhancements to the package commands"
+image: /img/kapp-controller.svg
+tags: ['carvel', 'kapp-controller', 'gitops', 'apps', 'kctrl', 'cli']
+---
+
+Since the previous release of `kctrl`, we have been iterating over how we can help users take a closer look at what App CRs are doing on the cluster. This includes ones authored by users and those that are created as a result of package installations.
```

Comment:
nit: This includes APP CRs authored by users as well as created by package installations.

Reply:
Adopted